__tests__/TelInput.test.js

```diff
@@ -478,6 +478,20 @@ describe('TelInput', function () { // eslint-disable-line func-names
 
       expect(subject.find(TelInput).props().placeholder).toBe('Your phone');
     });
+
+    it('should change input placeholder on customPlaceholder prop change', () => {
+      const subject = this.makeSubject();
+
```

Comment:
Add:

```js
    requests[0].respond(200,
      { 'Content-Type': 'text/javascript' },
      libphonenumberUtils);
    window.eval(getScript().text);
```

Because the `setState` in `updatePlaceholder` will execute only when `window.intlTelInputUtils` is not undefined.

Reply:
Should be good, thanks!